action/action.go

```diff
@@ -53,13 +52,16 @@ func New(v string) *Action {
 	cfg.ImportFunc = askForKeyImport
 	cfg.FsckFunc = askForConfirmation
 	cfg.Version = v
+
+	// write the config to disk for persisten
+	writeConfig(cfg)
```

Comment:
We discussed that prior to release already and agreed to only write the config to disk when running "gopass init" or changing any config values, so I'd have to ask to to remove this line.